docs/internal-documentation/maintenance/TechnicalMaintenancePlan.md

```diff
@@ -157,7 +157,9 @@ The software that runs the ASN Lookup service is rbldnsd, and the software itsel
 
 ### Github
 
-The Github repository is a service run by the GitHub company. However, our repository for the Zonemaster software is belongning to the "dotse"-organization within GitHub, and the "dotse" organization is being managed by people from IIS. The Zonemaster repository is a separate source code versioning repository under "dotse", and is being separately managed by its own repository managers.
+The Github repository is a service run by the GitHub company. However, our repositories for the 
+Zonemaster software belong to the Zonemaster organization within GitHub, and the Zonemaster 
+organization is being managed by people from Afnic and IIS. 
```

Comment:
This sounds almost like we're a part of the Github company.

Reply:
The text needs updates, but that is out of scope of this PR.